puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/hammerCliKatelloNightlyRelease.groovy

```diff
@@ -12,19 +16,21 @@ pipeline {
         stage ("Build Gem") {
             steps {
                 dir('hammer_cli_katello') {
+                    configureRVM(env.RUBY)
                     git url: "https://github.com/Katello/hammer-cli-katello", branch: 'master'
-                    sh "gem build hammer_cli_katello.gemspec"
-                    archiveArtifacts artifacts '*.gem'
+                    withRVM(["gem build hammer_cli_katello.gemspec"], env.RUBY)
```

Comment:
Does this need an RVM environment configured? Just curious if that's overhead we are incurring or required to `gem build`

Reply:
I found that without configuring an RVM environment, I didn't have access to the `gem` command.

Reply:
Odd, the nodes should have Ruby installed. And that was working if I look randomly at http://ci.theforeman.org/job/hammer-cli-katello-nightly-release/66/console

Reply:
Perhaps they have the `ruby` package but not always `rubygems`? We can just ensure that via the puppet `slave` module with `ensure_packages`.